core/web/presenters/ocr_keys.go

```diff
@@ -57,9 +58,9 @@ func NewOCR2KeysBundleResource(key ocr2key.KeyBundle) *OCR2KeysBundleResource {
 	return &OCR2KeysBundleResource{
 		JAID:              NewJAID(key.ID()),
 		ChainType:         string(key.ChainType()),
-		OnchainPublicKey:  key.OnChainPublicKey(),
-		OffChainPublicKey: hex.EncodeToString(key.OffchainPublicKey()),
-		ConfigPublicKey:   hex.EncodeToString(configPublic[:]),
+		OnchainPublicKey:  fmt.Sprintf("ocr2on_%s_%s", key.ChainType(), key.OnChainPublicKey()),
+		OffChainPublicKey: fmt.Sprintf("ocr2off_%s_%s", key.ChainType(), hex.EncodeToString(key.OffchainPublicKey())),
+		ConfigPublicKey:   fmt.Sprintf("ocr2cfg_%s_%s", key.ChainType(), hex.EncodeToString(configPublic[:])),
```

Comment:
Neither of these is chain-specific, so we could in principle get rid of the ChainType(). WDYT?

Reply:
I'm fine with that. These pub keys will both _belong_ to a key bundle that does have a specific chain type. So don't think it's necessarily misleading. But I have no strong preference. We should just do whatever is easier / simpler for node ops.

Reply:
it might feel awkward to have the keys print out read... WDYT? @kaleofduty 

```
Type:             solana
On-chain pubkey:  ocr2on_solana_f627...
Off-chain pubkey: ocr2off_2b87...
Config pubkey:    ocr2cfg_ce6e...
```

Reply:
just my two cents, but i also agree it's a bit awkward

i like the idea of dropping `_solana_` altogether since the keys are part of a bundle with the type listed?

Reply:
i think keeping it might be better?
matching it with the on-chain pubkey format, and also if the keys ever get separated from the bundle, there's indication of what chain it was used for?

Reply:
+1 on keeping the prefix.